frontend/app/components/organisms/ViewMeeting/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react'
+import styles from '../../../../styles/ViewMeeting.module.scss';
+
+type ViewMeetingDetailsProps = {
+  title: string;
+  setting: string;
+  date: string;
+  time: string;
+  recurrence?: string;
+  calendar: string;
+  location: string;
+  zoomAccount: number;
+  zoomLink: string;
+  pandaDocsName: string;
+  pandaDocsSize: string;
+  pandaDocsLink: string;
+  onBack: () => void;
+  onEdit: () => void;
+  onDelete: () => void;
+};
+
+const ViewMeetingDetails: React.FC<ViewMeetingDetailsProps> = ({
+  title,
+  setting,
+  date,
+  time,
+  recurrence,
+  calendar,
+  location,
+  zoomAccount,
+  zoomLink,
+  pandaDocsName,
+  pandaDocsSize,
+  pandaDocsLink,
+  onBack,
+  onEdit,
+  onDelete,
+}) => {
+  return (
+    <div className={styles.meetingDetails}>
+      <div className={styles.header}>
+        <button className={styles.backButton} onClick={onBack}>←</button>
+        <h1>{title}</h1>
+        <span className={styles.settingLabel}>{setting}</span>
+        <div className={styles.moreOptions}>
+          <button>⋮</button>
+          <div className={styles.optionsMenu}>
+            <button onClick={onEdit}>Edit Meeting</button>
+            <button onClick={onDelete}>Delete Meeting</button>
+          </div>
+        </div>
+      </div>
+      <div className={styles.details}>
+        <p style={{ color: 'gray' }}>📅 {date}</p>
```

Comment:
For these icons, rather than using emojis, try to use some icons from an external library to match the Figma designs